src/devtools/client/debugger/src/components/Editor/ColumnBreakpoints.js

```diff
@@ -24,14 +39,20 @@ class ColumnBreakpoints extends Component {
     }
 
     let breakpoints;
+
+    // We have to go through some extra steps because of a CodeMirror bug:
+    // https://github.com/codemirror/CodeMirror/issues/6460
+    const sortedColumnBreakpoints = this.getColumnBreakpoints();
+
     editor.codeMirror.operation(() => {
-      breakpoints = columnBreakpoints.map(breakpoint => (
+      breakpoints = sortedColumnBreakpoints.map((breakpoint, i) => (
         <ColumnBreakpoint
           cx={cx}
           key={makeBreakpointId(breakpoint.location)}
           columnBreakpoint={breakpoint}
           editor={editor}
           source={selectedSource}
+          insertAt={this.isLastBreakpoint(i) ? null : i}
```

Comment:
CodeMirror allows you to specify a position for a particular widget using `insertAt`, so we pass in a number here that allows us to order the breakpoint panel widgets to prevent shuffling. 

But there's an `insertAt` bug where CodeMirror won't allow you to add a widget to the last position of existing widgets (e.g. adding a widget to the end of three existing widgets). This is a workaround for that bug.

https://github.com/codemirror/CodeMirror/issues/6460

Reply:
The maintainer fixed it! I'll fix this tomorrow morning.

Reply:
Nice!